boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -90,12 +90,14 @@ list(APPEND FREELDR_COMMON_SOURCE
 
 if(ARCH STREQUAL "i386")
     list(APPEND FREELDR_COMMON_ASM_SOURCE
+        arch/i386/drvmap.S
+        # arch/i386/mb.S
```

Comment:
It appears that this file is a left-over from good-ol' times and can actually be removed completely from compilation.
Please let me know if you think it should be used somewhere...